src/storage/src/source/source_reader_pipeline.rs

```diff
@@ -928,22 +489,11 @@ where
         if is_active_worker {
             if let Some(status_shard) = storage_metadata.status_shard {
                 info!("Health for source {source_id} being written to {status_shard}");
-                write_to_persist(
```

Comment:
why was this block removed? Because it now implicitly happens in the loop below?